x-pack/plugins/observability_solution/inventory/common/entities.ts

```diff
@@ -4,7 +4,8 @@
  * 2.0; you may not use this file except in compliance with the Elastic License
  * 2.0.
  */
-import { ENTITY_LATEST, entitiesAliasPattern } from '@kbn/entities-schema';
+import { z } from '@kbn/zod';
```

Comment:
Adding zod,  inventory now has both io-ts and Zod. For consistency, could we stick to just one?

Reply:
I think the idea is to move to `zod`. IMO it's less harmful to use `zod` than to have a duplicate definition of [`entity_latest`](typeof entityLatestSchema), which currently is defined with `zod`

Reply:
Should we create a tech depth ticket then to unify this and use one tool?

Reply:
Yeah. I think so. I'll do that :)